kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -81,44 +81,71 @@
       </KGrid>
 
       <h2>{{ $tr('chooseExercises') }}</h2>
+      <div v-if="!showChannels">
+        <br>
+        <ContentCardList 
+          :contentList="bookmarkNodes"
+          :contentHasCheckbox="contentHasCheckbox"
+          :contentCardMessage="() => {return ''}"
```

Comment:
Props don't have to be bound, but can also be constant values. So to simplify this, you can just set the value to the empty string. It could be rewritten as:
`contentCardMessage=''` 
(note that the : at the front of the prop has been removed)

Reply:
Yes, that makes sense, that was also my first approach `contentCardMessage=""` but then i receive this error
![Screenshot from 2021-08-16 15-00-18](https://user-images.githubusercontent.com/52789571/129560622-7fd9284f-518c-4ec2-9248-d8d305f48945.png) 
and that is why i passed it as a function.

Reply:
yeah, looks like in this case the prop requires a function which returns the value:

https://github.com/learningequality/kolibri/blob/b46ec1b5b0376dcd8fc86fb9628e32cb091042e5/kolibri/plugins/coach/assets/src/views/plan/LessonResourceSelectionPage/ContentCardList.vue#L113-L117

Reply:
Ooof - that seems like an idiosyncratic choice, but probably now is not the time to change it.